pages/desktop/consumer_pages/account_settings.py

```diff
@@ -112,6 +114,17 @@ def edit_language(self, option_value):
         select = Select(element)
         select.select_by_value(option_value)
 
+    def click_enable_recommendations_button(self):
```

Comment:
This will toggle the checkbox, which is fine given that you're asserting the state immediately after. The name of this method could be misleading though - it's suggesting that you're enabling (you could be disabling) by clicking a button (it's a checkbox). I'd suggest `toggle_recommendations`.

Alternatively, we could have two methods: `enable_recommendations` and `disable_recommendations` that would give better readability for the test. These methods would internally assert that the checkbox state before interacting with it.